api/docker-entrypoint.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+info()
+{
+    echo "Info: $*"
+}
+
+warn()
+{
+    echo "Warning: $*"
+}
+
+fatal()
+{
+    echo "Fatal: $*"
+}
+
+get_couchdb_url()
+{
+  echo "http://$COUCHDB_USER:$COUCHDB_PASSWORD@$COUCHDB_SERVICE_NAME:5985"
+}
+
+wait_for_couchdb()
+{
+  local n=60
+
+  while [ "$n" -gt 0 ]; do
+    curl "`get_couchdb_url`" &>/dev/null
+    [ "$?" -eq 0 ] && return 0
+    sleep 3
+    n=$[$n-1]
```

Comment:
The sleep time and total time are dependent on each other, but that's one kind of works as a side effect of the other. Please give these variables more descriptive names and modify the logic a bit so the sleep interval can be changed without breaking the total time.